commands/set_channel.js

```diff
@@ -0,0 +1,31 @@
+const { SlashCommandBuilder, PermissionsBitField } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('set_channel')
+		.setDescription('Set a specific channel for the bot to use.')
+		.setDMPermission(false)
+        .setDefaultMemberPermissions(PermissionsBitField.Flags.Administrator)
+        .addIntegerOption(option =>
+			option.setName('mistakes_allowed')
+				.setDescription('Number of mistakes allowed')
+				.setRequired(true)),
+	async execute(interaction, client) {
+        try {
+            await client.sequelize.models.Channel.create({ 
+                mistakesAllowed: interaction.options.getInteger('mistakes_allowed'), 
+                id: interaction.channel.id,
+                server: interaction.guild.id
+                });
+
+            await interaction.reply({ content: 'Channel is set successfully!', ephemeral: true });
+            
+            // TODO: Call to function OnCreate()
+            
+        } catch (error) {
+            console.error(error);
+            await interaction.reply({ content: 'Couldn\'t set channel :c', ephemeral: true });
+        }
```

Comment:
This should happen in OnCreate. This command should only test if the channel is already registered, or can be registered at all. The latter is quite tricky. It requires checking channel permissions. I think it's not required at the moment.